receiver/datadogreceiver/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/datadogreceiver
+
+go 1.14
```

Comment:
That should be bumped to match was is set within the collector unless that is a reason why it should be this.

Reply:
Moved to 1.17